### ISSUE TITLE
Pauze on inline links instead of combining them

### ISSUE THREAD

**Issue by Veyfeyken** [nvaccess/nvda#5605]
Demo: http://gijs.anysurfer.be/test/temp.html 
There's no stop when navigating with the arrow-keys between links. Is this a bug or a feature?
NVDA combines the links. This can be confusing. The only way to put focus on the second link is to use the tab key. Jaws and VoiceOver stop at each link. Wouldn't that be the expected behaviour?

**Comment by jcsteh** [comment]
NVDA's default behaviour is to present the page more closely to the way it appears on the screen; i.e. inline elements are on the same line. This makes it more pleasant to read text with a lot of embedded links, easier to grasp certain forms (e.g. a phone or credit card number divided across several fields) and easier to skip horizontal navigation bars, just to give a few examples.

You can focus other links on the same line either by moving with cursor keys (e.g. next word), tab, or "k" to move to the next link. You can also disable this behaviour by turning off screen layout; see the User Guide for details.

In short, what you're seeing is the intended default behaviour.

**Comment by bhavyashah** [comment]
Dear Jamie,
It is an arguable issue whether the default behaviour should be screen
layout disabled or not. For presentation purposes in a
mainstream/sighted way, screen layout would be useful, and in specific
cases screen layout is toggled by me to on, but as an efficient screen
reader user, I would like to believe that most users primarily disable
screen layout, only enabling it in certain circumstances where it
improves navigability.
It would be interesting to know views of other NVDA users (trainers
particularly) too.
Thanks.

On 12/14/15, James Teh <notifications@github.com> wrote:
> Closed #5605.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/5605#event-490832018


-- 
Warm Regards
Bhavya Shah
Using NVDA (Non Visual Desktop Access) free and open source screen
reader for Microsoft Windows
To download a copy of the free screen reader NVDA, please visit
http://www.nvaccess.org/
Using Google Talkback on Motorolla G second generation Lollipop 5.0.2
Reach me through the following means:
Mobile: +91 7506221750
E-mail id: bhavya.shah125@gmail.com
Skype id : bhavya.09
